stdlib/char.gr

```diff
@@ -2,10 +2,11 @@ import WasmI32 from "runtime/unsafe/wasmi32"
 import Memory from "runtime/unsafe/memory"
 import Errors from "runtime/unsafe/errors"
 import { tagSimpleNumber, allocateChar, allocateString } from "runtime/dataStructures"
-import foreign wasm throwError: (WasmI32, WasmI32, WasmI32) -> a from "grainRuntime"
 
 export *
 
+exception MalformedUtf8
```

Comment:
Should this have a printer?

Reply:
I thought the default printer was fine, but I suppose I could see a printer for it.